### PR TITLE
CNV-27107: RN instance type

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.14 RN

Issue: [CNV-27107](https://issues.redhat.com//browse/CNV-27107)

Link to docs preview: https://62274--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes.html#virt-4-14-technology-preview

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Comment by RoniKishner** [comment]
Looks good, QE ack

**Review by skrthomas — APPROVED** [review]
Merge review lgtm

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -141,6 +139,7 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
+* You can now use xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-instancetype_virt-create-vms[common instance types] to install and edit customized instance types and preferences to create a VM.
```
[the remark this answers is not in the document]
Comment (RoniKishner) [reply virt/release_notes/virt-4-14-release-notes.adoc]: Ignore this, it was on the previous version.

**Pull request merged**